res/css/views/beta/_BetaCard.pcss

```diff
@@ -15,7 +15,6 @@ limitations under the License.
 */
 
 .mx_BetaCard {
-    margin-bottom: $spacing-20;
```

Comment:
could you help me understand why this (and the `last-child` below) are no longer needed?

Reply:
Because the grid `.mx_SettingsSection_subSections` defines 32px row gap between elements inside it by replacing the `margin-bottom`. Technically, it is very clear why you no longer need margin-bottom here, though I do not know if increasing the value (from `spacing-20` to 32px) is expected.

Reply:
Like luixxiul says, spacing of the beta cards is now handled by their container. Change in the exact spacing value adheres the cards to the system for settings.